BackendServices/AuxiliaryServices/WebUtils/OHS/Statistic.cs

```diff
@@ -45,5 +45,46 @@ public class Statistic
             else
                 return JaminProcessor.JaminFormat("{ [\"status\"] = \"fail\" }", 0);
         }
+
+
+        public static string? Tracker(byte[] PostData, string ContentType)
+        {
+            string? dataforohs = null;
+
+            string? boundary = HTTPUtils.ExtractBoundary(ContentType);
+
+            if (boundary != null)
+            {
+                using (MemoryStream ms = new(PostData))
+                {
+                    var data = MultipartFormDataParser.Parse(ms, boundary);
+                    try
+                    {
+                        LoggerAccessor.LogInfo($"[OHS] : Client Version - {data.GetParameterValue("version")}");
+                    }
+                    catch (Exception)
+                    {
+                        // Not Important.
+                    }
+                    try
+                    {
+                        //dataforohs = JaminProcessor.JaminDeFormat(data.GetParameterValue("data"), true, 0);
```

Comment:
Maybe print the body? Maybe this heatmap can be usefull gor the admin.

Reply:
well right now its commented out entirely for E3 2010 so we don't need it right now. As this caused the video loading problems agent... Too many http request. So not really needed.

Reply:
Otherwise this will be more issue than its worth. If i knew it caused video loading problems i wouldn't have added it initially tbh.

Reply:
Ok so it's fine.

Reply:
Yeah, i will still add the logger for it, but otherwise i added bool check in E3 2010 lua for this in case you still want to try it. It is in HHQ fixed sdat.